next-sitemap.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next-sitemap').IConfig} */
```

Comment:
Do we need this file regarding the task?

Reply:
removed